python-project-template/.github/workflows/{% if include_benchmarks %}asv-main.yml{% endif %}.jinja

```diff
@@ -12,20 +12,14 @@ env:
   PYTHON_VERSION: "3.10"
   WORKING_DIR: {% raw %}${{ github.workspace }}{% endraw %}/benchmarks
 
-jobs:
-
-  consecutiveness:
-    runs-on: ubuntu-latest
+concurrency:
+  group: {% raw %}${{ github.workflow }}-${{ github.ref }}{% endraw %}
```

Comment:
I don't think we need `github.ref` here: it is unlikely to have to jobs with the same git reference to run simultaneously